integration-tests/tests/api/artifacts-cdn.spec.ts

```diff
@@ -662,7 +662,7 @@ describe('CDN token', () => {
     expect(deleteResult).toEqual(
       expect.arrayContaining([
         expect.objectContaining({
-          message: `No access (reason: "Missing target:settings permission")`,
+          message: `No access (reason: "Missing permission for performing 'cdnAccessToken:delete' on resource")`,
```

Comment:
Since we now have granular permissions the messages changed slightly. We do not have any logic that depends on the message contents, except integration test fixtures.